src/redux/ConfigureStore.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+// eslint-disable-next-line
+import categoryReducer from "./categories/categories";
```

Comment:
- [ ] As requested by @Gambit142 the correct path should be:
"./Categories/categories". In your path for the import statement, you are using lowercase c instead of uppercase C for the Categories directory word. Please 🙏🏽 correct the path to avoid compilation errors.

- [ ] As requested by @Gambit142 kindly remove the Eslint-disable statement from this file. The Eslint helps us write clean, maintainable, and scalable codes. Using Eslint-disable statements defeats this purpose.